app/controllers/account/settings.js

```diff
@@ -56,6 +58,17 @@ export default Controller.extend({
     return repositories.filter(repo => !repo.emailSubscribed);
   }),
 
+  userHasNoEmails: computed('auth.currentUser.emails', function () {
+    if (!this.auth.currentUser.emails || this.auth.currentUser.emails.length === 0)
```

Comment:
why not return !this.auth.currentUser.emails || this.auth.currentUser.emails.length === 0